content/en/tracing/setup/nodejs.md

```diff
@@ -124,6 +133,7 @@ For details about how to how to toggle and configure plugins, check out the [API
 | [mysql2][26]           | `>=1`    | Fully Supported |                                                  |
 | [pg][27]               | `>=4`    | Fully Supported | Supports `pg-native` when used with `pg`         |
 | [redis][28]            | `>=0.12` | Fully Supported |                                                  |
+| [tedious][46]          | `>=1`    | Fully Supported | SQL Server driver for `mssql` and `sequelize`    |
```

Comment:
```suggestion
| [tedious][46]          | `>=1`    | Fully Supported | SQL Server driver for `mssql` and `sequelize`. Requires `dd-trace-js` v0.13.0+     |
```

Reply:
As discussed, let's leave this out for now since it would be difficult to track the exact library version support by tracer version.